autoware_launch/config/planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/obstacle_stop_planner.param.yaml

```diff
@@ -4,6 +4,8 @@
     lowpass_gain: 0.9                        # gain parameter for low pass filter [-]
     max_velocity: 20.0                       # max velocity [m/s]
     enable_slow_down: False                  # whether to use slow down planner [-]
+    enable_z_axis_obstacle_filtering: True
+    z_axis_filtering_buffer: 0.0
```

Comment:
Please add some short descriptions of the new parameters, including the unit of the buffer parameter.

Reply:
I've added a short description of the new parameters